### PR TITLE
Telescope: handle multiple selections

### PR DESCRIPTION
**Pull request by jghauser**
~TODO: handle creating multiple notes when trying to open notes which do not exist yet~ [DONE]

### CODE REVIEW

File: lua/telescope/_extensions/papis/actions.lua [hunk 1]
```diff
@@ -11,50 +11,108 @@ local db = require("papis.sqlite-wrapper")
...
   actions.close(prompt_bufnr)
-  vim.api.nvim_put({ entry }, "", false, true)
+  local string_to_insert = ""
+  if vim.tbl_isempty(multi) then
+    local ref = string.format(format_string, action_state.get_selected_entry().id.ref)
+    string_to_insert = ref
+  else
+    for _, entry in pairs(multi) do
+      local ref = string.format(format_string, entry.id.ref)
+      string_to_insert = string_to_insert .. ref .. " "
```
Comment (maxmahlke) [inline lua/telescope/_extensions/papis/actions.lua:36]: The string connecting the reference keys (`" "` here)  could be another configuration parameter. I only use `papis.nvim` with `tex` files and I need to connect them via `,`

Comment (jghauser) [reply lua/telescope/_extensions/papis/actions.lua:36]: Good point! I'll make sure to make it configurable and try to see what would be sensible defaults for the various filetypes we configure in `cite_formats`. I might in fact add it to that option :)

**Comment by jghauser** [comment]
hey @maxmahlke, if you have the time, it would be great if you could take another look. i implemented what you asked for (configurable strings between refs) and also managed to sort out the other issues left with opening multiple files (especially notes). note that there are breaking changes concerning the `citation_formats`, `format_notes`, and `format_references` options (in case you're not using the defaults). The readme should reflect these changes.

**Comment by jghauser** [comment]
TODO: allow inserting complete citations. E.g. rather than `@SomeDude2011`, do `[@SomeDude2011]`. This should now be possible because we correctly handle multiple citations. However, I'm not sure what's the best way to handle this given that the old way is still useful at times.

**Comment by maxmahlke** [comment]
Hi @jghauser! I tested it briefly and it worked well for me in a `tex` file! Also a nice solution with the settings, it's quite flexible now. I prefer not having spaces in my `\cite` commands, so I set `separator_str=','`. I'll keep using the branch and let you know if I find a bug.

**Comment by jghauser** [comment]
@maxmahlke, thank you so much! :heart: Do you think that the separator with space is more common for tex files? I probably just did a super short search and saw someone not using a space and made that the default. Happy to change the default though!

**Comment by jghauser** [comment]
Thanks a lot @maxmahlke, I've merged this now as I think it's ready. :crossed_fingers: Let me know if you still come across some issues :)

**Pull request merged**